docs/developers/caching.md

```diff
@@ -0,0 +1,13 @@
+---
+outline: deep
+---
+
+# Caching
+
+## Cache-based requests
+
+The Datahub leverages OGC-client's built-in caching mechanism to optimize data fetching for features, CSV, XLS, GeoJSON, GML files, and more.
```

Comment:
Strictly speaking, the OGC client cache concerns data served from OGC services, but I think we also do caching for data like CSV and XLS in the DataFetcher, right? In any case what you wrote is understandable I think.

Reply:
indeed, we also directly use the useCache utility function from OGC-C to cache some of our requests, especially the ones for fetching files.